client/jetpack-cloud/sections/pricing/header/index.tsx

```diff
@@ -44,10 +51,23 @@ const Header: React.FC = () => {
 			'Get everything your site needs, in one package — so you can focus on your business.'
 		);
 
+	// Black Friday 2020 promotion; runs Nov 20-30 automatically. Safe to remove after Dec 1
+	const promoStartDate = moment( '2020-11-20', 'YYYY-MM-DD h:mm:ss a' );
+	const promoEndDate = moment( '2020-11-30 11:59:59 pm', 'YYYY-MM-DD h:mm:ss a' );
+	const today = moment();
+
+	// Use query param `?bf=true` to preview the banner outside of Black Friday: https://cloud.jetpack.com/pricing?bf=true
+	const hasPromoQueryParam = urlQueryArgs?.bf && urlQueryArgs.bf === 'true' ? true : false;
```

Comment:
Non-blocking tip: This could be re-written as:

```js
const hasPromoQueryParam = urlQueryArgs?.bf === 'true';
```

Reply:
Ahh yes, that's better, thank you. 👍